tutorials/setup-your-development-environment.md

```diff
@@ -0,0 +1,67 @@
+# Setup your development environment
+
+The following section walks you through the necessary steps to setup your environment.
+It is recommended to spin up a clean Ubuntu VM for these steps as it is always good to perform experiments in an isolated development environment.
+The steps outlined [here](https://juju.is/docs/sdk/set-up-your-development-environment#heading--set-up-an-ubuntu-vm-with-multipass) explain how to do this.
+
+## Create a project directory
+
+Create a new project directory with
+```bash
+mkdir ~/go-dqlite-demo
+cd ~/go-dqlite-demo
+```
+
+This will create a new folder called `go-dqlite-demo` in your home directory and opens it. All project files will be put in this directory.
+
+## Install the Dqlite tooling
+
+On a debian-based system you can get the latest development release from [Dqlite's development PPA](https://launchpad.net/~dqlite/+archive/ubuntu/dev). First add the PPA repository:
+
+```bash
+sudo add-apt-repo ppa:dqlite/dev
+sudo apt update
+```
+
+then install the library with 
+
+```bash
+sudo apt install libdqlite-dev
+```
+
+[note type="important" status="Info"]
+If you are not using a debian-based system you’ll need to build libdqlite from source. See [this guide](https://github.com/canonical/dqlite#build) on instructions on how to do this.
+[/note]
+
+# Install Go 
+
+Download and install the latest version of Go following the steps outlined here: https://go.dev/doc/install.
+You can verify that everything is installed correctly by typing the following command in your terminal:
+
+```bash
+> go version
+go version go1.21.1 linux/amd64
```

Comment:
We should give some information about what we expect to see here. "not found" would probably be bad. ;)
Maybe say that if the version is >something, it's good?

Reply:
There are no real version constrains. If the user follows the linked tutorial, the version should be pretty recent. 

I added one sentence that the user should check if the version output matches the one that was installed during the installation step.